src/lib/components/header.svelte

```diff
@@ -1,16 +1,46 @@
 <script lang="ts">
   import ProfilePicture from './profile-picture.svelte';
   import { page } from '$app/stores';
+  import { onMount } from 'svelte';
   export let username: string;
 
   const navItems = [{ name: 'Docs', path: '/docs' }];
+  let closeHamburgerMenu: () => void;
+
+  onMount(() => {
+    const burger = document.querySelector('.navbar-burger');
```

Comment:
I think you can use `bind:this` to bind an HTMLElement. You don't need a querySelector here 🤔

Reply:
Sure, did the changes.